server/middleware/verifyToken.js

```diff
@@ -33,15 +33,14 @@ const verifyToken = {
             id: authData.id
           }
         })
-        .then((user) => {
+        .then(() =>
           // Token is valid but user does not exist
```

Comment:
Hi jerry, great work as usual. Please take care of the commented out codes again to keep your codes at it's best.